content/english/blog/post-20210726-1.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Deployment of an OSCAR cluster in the EGI Federated Cloud"
+date: 2021-07-25T13:00:00+01:00
+# post image
+image: "images/blog/post-20210726-1/portada_blog_oscar_synergy.png"
+# post type (regular/featured)
+type: "featured"
+# meta description
+description: "This is meta description"
+# post draft
+draft: false
+---
+
+Here we present a step by step guide to help you deploy an OSCAR cluster using EGI credentials in the [EOSC-Synergy VO](https://infra.eosc-synergy.eu/vos/#synergy). We are using the [IM Dashboard](https://appsgrycap.i3m.upv.es:31443/im-dashboard/login), a tool developed by the [GRyCAP](https://www.grycap.upv.es/) research group at the [Universitat Politècnica de València](https://www.upv.es/) to facilitate the deployment of infrastructures in a lot of cloud providers. Alternatively, you can follow our YouTube video, at the end of the post.
+
+**Step 1:** Go the the [IM Dashboard](https://appsgrycap.i3m.upv.es:31443/im-dashboard/login) and click the button "Login with EGI-Check-in".
+
+![Welcome to the IM Dashboard](../../images/blog/post-20210726-1/001.png)
+
+**Step 2:** EGI Check-in supports multiple identity providers, select one and access the IM Dashboard.
+
+![EGI Check-in](../../images/blog/post-20210726-1/002.png)
+
+**Step 3:** Go to the top right corner and click on your username to access "Cloud Credentials".
+
+![Cloud Credentials](../../images/blog/post-20210726-1/003.png)
+
+**Step 4:** Click "New Credential" button and select EGI.
+
+![New EGI Credential](../../images/blog/post-20210726-1/005.png)
+
+**Step 5:** Write a new name for your new credential, select the EOSC-Synergy VO, and select the desired provider (the dropdown list can take a few seconds to load). Click "Add" to finish.
+
+![Synergy VO Credential](../../images/blog/post-20210726-1/006.png)
+
+**Step 6:** Return to the IM Dashboard by clicking on "IM Dashboard" at the top left corner. Go to "Launch a OSCAR Virtual Cluster" and click "Configure".
```

Comment:
It should read "Launch *an* OSCAR Virtual Cluster". We can keep the screenshots and the video but the entry in the IM dashboard should be updated // cc @micafer

Reply:
Ok, text fixed.